SpatialGDK/Source/SpatialGDKEditor/Private/Utils/LaunchConfigEditorLayoutDetails.cpp

```diff
@@ -3,25 +3,14 @@
 #include "Utils/LaunchConfigEditorLayoutDetails.h"
 
 #include "SpatialGDKSettings.h"
-#include "Utils/LaunchConfigEditor.h"
+#include "Utils/LaunchConfigurationEditor.h"
 #include "DetailLayoutBuilder.h"
 
 TSharedRef<IDetailCustomization> FLaunchConfigEditorLayoutDetails::MakeInstance()
 {
 	return MakeShareable(new FLaunchConfigEditorLayoutDetails);
 }
 
-void FLaunchConfigEditorLayoutDetails::ForceRefreshLayout()
-{
-	if (MyLayout != nullptr)
-	{
-		TArray<TWeakObjectPtr<UObject>> Objects;
-		MyLayout->GetObjectsBeingCustomized(Objects);
-		ULaunchConfigurationEditor* Editor = Objects.Num() > 0 ? Cast<ULaunchConfigurationEditor>(Objects[0].Get()) : nullptr;
-		MyLayout->ForceRefreshDetails();
-	}
-}
-
 void FLaunchConfigEditorLayoutDetails::CustomizeDetails(IDetailLayoutBuilder& DetailBuilder)
```

Comment:
It looks like this customization doesn't actually do anything. We should just remove this class.

Reply:
it's used here. but I'm not sure if it really needed. 
PropertyModule.RegisterCustomClassLayout(ULaunchConfigurationEditor::StaticClass()->GetFName(), FOnGetDetailCustomizationInstance::CreateStatic(&FLaunchConfigEditorLayoutDetails::MakeInstance));

If not, I can remove it.

Reply:
Yes, I think it's not needed. I tried removing that line and the behavior is not affected. I believe `IDetailCustomization` is usually used to add elements to the settings section, but here nothing is actually being added.

Reply:
The customization was needed until the ServerWorkerMap was removed from the settings. You can remove it now.